h2o-docs/src/product/data-munging/sortcolumn.rst

```diff
@@ -1,74 +1,20 @@
-Sorting Columns
----------------
+Sort columns
+============
 
-Use the ``sort`` function in Python or the ``arrange`` function in R to create a new frame that is sorted by column(s) in ascending (default) or descending order. Note that when using ``sort``, the original frame cannot contain any string columns. 
+Use the ``sort`` function in Python or the ``arrange`` function in R to create a new frame that is sorted by column(s) in ascending (default) or descending order. 
 
-If only one column is specified in the sort, then the final results are sorted according to that one single column either in ascending (default) or in descending order. However, if you specify more than one column in the sort, then H2O performs as described below:
+.. note::
+	
+	When using ``sort``, the original frame cannot contain any string columns. 
```

Comment:
Please remove 

When using "sort", the original frame cannot contain any string columns.  

I fixed this long time agin.